scripts/test-simple-setup.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Sleeping for a bit to ensure db"
+
+sleep 10
```

Comment:
fine for now, maybe leave a TODO for this to use the health endpoint in the future.